package.json

```diff
@@ -1,24 +1,29 @@
 {
-  "name": "my-restaurant-page",
+  "name": "restaurant",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/tmampa/my-restaurant-page.git"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server": "webpack serve"
```

Comment:
- [ ] Within the scripts, add the necessary run common commands like build, development.
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```